lib/systems/examples.nix

```diff
@@ -98,20 +98,14 @@ rec {
   # can execute on 32bit chip
   mips-linux-gnu                = { config = "mips-linux-gnu";                } // platforms.gcc_mips32r2_o32;
   mipsel-linux-gnu              = { config = "mipsel-linux-gnu";              } // platforms.gcc_mips32r2_o32;
-  mipsisa32r6-linux-gnu         = { config = "mipsisa32r6-linux-gnu";         } // platforms.gcc_mips32r6_o32;
-  mipsisa32r6el-linux-gnu       = { config = "mipsisa32r6el-linux-gnu";       } // platforms.gcc_mips32r6_o32;
 
   # require 64bit chip (for more registers, 64-bit floating point, 64-bit "long long") but use 32bit pointers
   mips64-linux-gnuabin32        = { config = "mips64-linux-gnuabin32";        } // platforms.gcc_mips64r2_n32;
   mips64el-linux-gnuabin32      = { config = "mips64el-linux-gnuabin32";      } // platforms.gcc_mips64r2_n32;
```

Comment:
Gcc does not know about this? https://hercules-ci.com/accounts/github/nix-community/derivations/%2Fnix%2Fstore%2F4ggih217yc0s4i7xs1wz3rdf1y71kj1c-mipsel-linux-gnu-stage-static-gcc-debug-10.3.0.drv/log?via-job=1df40d2a-d53d-4f4d-a872-9f43d9f87a85

Reply:
clang also fails:

nix-repl> :b pkgsCross.mips64-linux-gnuabin32.buildPackages.clang
error: build of '/nix/store/zjwxrsx8vjmhs1yqs8jy8973zcrkdf8f-compiler-rt-libc-mips64-linux-gnuabin32-11.1.0.drv' on 'ssh-ng://nix@ryan.r' failed: error: builder for '/nix/store/zjwxrsx8vjmhs1yqs8jy8973zcrkdf8f-compiler-rt-libc-mips64-linux-gnuabin32-11.1.0.drv' failed with exit code 2;
              last 10 log lines:
              > /build/compiler-rt-11.1.0.src/lib/asan/asan_interceptors_memintrinsics.h:53:10: warning: 'local_iovec.__sanitizer::__sanitizer_iovec::iov_base' may be used uninitialized in this function [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmaybe-uninitialized-Wmaybe-uninitialized8;;]
              >    53 |     uptr __offset = (uptr)(offset);                                     \
              >       |          ^~~~~~~~
              > In file included from /build/compiler-rt-11.1.0.src/lib/asan/asan_interceptors.cpp:172:
              > /build/compiler-rt-11.1.0.src/lib/asan/../sanitizer_common/sanitizer_common_interceptors.inc:3261:21: note: 'local_iovec.__sanitizer::__sanitizer_iovec::iov_base' was declared here
              >  3261 |   __sanitizer_iovec local_iovec;
              >       |                     ^~~~~~~~~~~
              > [ 80%] Built target RTAsan.mips64
              > [ 80%] Built target RTAsan_dynamic.mips64
              > make: *** [Makefile:136: all] Error 2
              For full logs, run 'nix log /nix/store/zjwxrsx8vjmhs1yqs8jy8973zcrkdf8f-compiler-rt-libc-mips64-linux-gnuabin32-11.1.0.drv'.
error: builder for '/nix/store/zjwxrsx8vjmhs1yqs8jy8973zcrkdf8f-compiler-rt-libc-mips64-linux-gnuabin32-11.1.0.drv' failed with exit code 1
error: 1 dependencies of derivation '/nix/store/ddvwbzlfnm0h06829gy3gzklqd4mg3kc-mips64-linux-gnuabin32-clang-wrapper-11.1.0.drv' failed to build

Reply:
fixed.